### DISCUSSION TITLE
Miniconda removed from MacOS 14

### DISCUSSION THREAD

**Discussion by DhanshreeA** [actions/virtual-environments#9748] [Installed software]
Our builds have started failing on MacOS images suddenly because some of our workflows rely on using Conda which stopped being shipped within MacOS images since v13 it appears. Curious to learn why this is the case

**Comment by Alexey-Ayupov** [comment]
Hello @DhanshreeA, currently we have no plans to install `Miniconda` back to images. Please use tasks like this https://github.com/marketplace/actions/setup-miniconda to setup `Miniconda` in runtime.

**Reply by DhanshreeA** [reply]
Hi @Alexey-Ayupov, thank you.
